lib/ex_vatcheck/xml.ex

```diff
@@ -2,9 +2,23 @@ defmodule ExVatcheck.Xml do
   @moduledoc false
   require SweetXml
 
-  def parse(doc, %SweetXpath{} = spec, subspec \\ []) when is_binary(doc) do
+  def parse(doc, spec, subspec \\ [])
+
+  def parse(doc, spec, subspec) when is_binary(doc) do
     doc
     |> SweetXml.parse(dtd: :none)
-    |> SweetXml.xpath(spec, subspec)
+    |> parse(spec, subspec)
+  end
+
+  def parse(doc, %SweetXpath{} = spec, []) when is_tuple(doc) do
+    SweetXml.xpath(doc, spec)
+  end
+
+  def parse(doc, %SweetXpath{} = spec, subspec) when is_tuple(doc) do
+    if SweetXml.xpath(doc, spec) do
```

Comment:
After receiving the newly namespaced XML response, SweetXml was breaking if the initial `%SweetXpath{}`  "container" element (in the event of a successful response, that was the `checkVatResponse` XML element).

This now verifies the container exists and is not `nil` before SweetXml tries to xpath further in.